src/streamlink/plugins/twitch.py

```diff
@@ -277,6 +357,13 @@ class Twitch(Plugin):
                        action="store_true",
                        help="""
         Do not open the stream if the target channel is hosting another channel.
+        """
+                       ),
+        PluginArgument("disable-ads",
+                       action="store_true",
```

Comment:
```
default=False,  
```

or force it `True` if it is stable enough

Reply:
No need to set a default value here for `action="store_true"`. I thought the value was `None`, but it's `False` if unset.

We can change the value once we know that it's working well enough...